doc/releases/changelog-dev.md

```diff
@@ -34,6 +34,9 @@
   Now it is more likely to generate optimal cuts for larger circuits.
   [(#2517)](https://github.com/PennyLaneAI/pennylane/pull/2517)
 
+* Added `PSWAP` operator.
```

Comment:
Hi @ankit27kh, as this is a new changelog, don't forget to add your name to the list of contributors in a new line! :slightly_smiling_face: